src/GUI/efpt30.core/Program.cs

```diff
@@ -10,6 +11,8 @@ static int Main(string[] args)
         {
             try
             {
+                Console.OutputEncoding = Encoding.UTF8;
```

Comment:
I am guessing that this is actually the only real change required to fix this issue?

Reply:
yes

Reply:
I have fixed the encoding issue in the latest daily, please try that out, so we can see what else needs to be fixed. Please share the output if possible, rather than screenshots.

Reply:
莫名其妙的问题\r\nDbContext:\r\nTestContext\r\nDebugView:\r\n\r\nDbContext:\r\nTest2Context\r\nDebugView:\r\n\r\n